### ISSUE TITLE
odyssey.js

### ISSUE THREAD

**Issue by webwurst** [codeformuenster/muenster-quiz#1]
`docker-compose.yml` erstellen und Start-Kommando in die Readme.

**Comment by rubit0** [comment]
Ich möchte mich gerne an dieser Quiz App beteiligen. Ich bin jedoch kein Javascript Experte, kann man auch ein anderes Framework bzw. Engine verwenden?

**Comment by webwurst** [comment]
Na klar, kein Problem. Uns erschien `odyssey.js` einfach als recht praktisch dafür, viel passiert ist seit dem ja noch nicht ;)
Womit würdest du gerne arbeiten?

**Comment by rubit0** [comment]
Ich möchte gerne Unity3d als "Frontend" benutzen. Damit kenne ich mich gut aus und kann zBsp. eine 3D Übersichtskarte von Münster implementieren und viele andere coole Dinge :)
Bzgl. der Quiz-fragen, sollen diese dynamisch von einem Server geladen?

**Comment by webwurst** [comment]
Das klingt auf jeden Fall ganz cool!
Ja, die Fragen sollten wohl dynamisch geladen werden. Aber für einen Prototypen kannst du natürlich auch erstmal ein paar statisch einfügen.

**Comment by rubit0** [comment]
Super, ich freue mich zum ersten mal am einem Open Source Projekt zu beteiligen!
Sobald ich meine Arbeit am Prototypen anfange können wir uns über Ideen, Umsetzung usw. austauschen.

**Comment by webwurst** [comment]
Ja, cool! Wenn du Zeit und Lust hast, dann schau auch gerne jetzt Samstag bei unserem [Tag der offenen Daten](http://codeformuenster.org/opendataday/) vorbei.

Außerdem haben wir einen Slack-Channel. Da ist grade die Anmeldung kaputt, aber ich kann dich per Mail einladen, wenn du willst.

**Comment by ubergesundheit** [comment]
:+1:

**Comment by rubit0** [comment]
Wow, danke für die Einladung zum Treffen, sieht sehr cool aus! Jedoch bin ich leider am dem Tag verhindert :-/
Am Slack Chanel würde ich gerne Teilnehmen, da die Kommunikation über den Issue tracker wohl in Spam ausartet. Meine mail: <gelöscht damit du keinen spam kriegst>

**Comment by ubergesundheit** [comment]
Jau Invite ist raus! Hab auch mal deine Adresse aus dem Kommentar entfernt damit du keinen Spam kriegst
